### PR TITLE
Cockpit 261

### PR DESCRIPTION
**Pull request by garrett**
Preview: https://garrett.github.io/cockpit-project.github.io/blog/cockpit-261.html

### CODE REVIEW

**Comment by garrett** [comment]
As usual, I generated the release notes with the following command:

```sh
_scripts/toolbox-ruby _scripts/generate-release-notes.rb -p
```

(There are other options available; run `-h` instead of `-p` to see them all. Most notable is that `-p` is for preview mode, which includes the pending PRs that haven't been merged and it includes links back to all the PRs with the release-note tag. There's also `-r` for when the projects have all been tagged for release, so versions in the release notes will match.)

Of course, this requires that the website toolbox is created first. If you haven't done that, run the following command before the command above:

```
_scripts/toolbox-create
```

**Comment by garrett** [comment]
Reassigning. I probably can't take care of the release notes this week. But I think you two should have everything necessary.

One little gotcha: You'll want to `git reset --hard origin/main` before running the script, as:
- the document's filename is based on the date (so new days mean new filenames, until the day of the actual release)
- we wouldn't want old graphics lingering
- graphics are named based on the release and PR's title (so if the PR's title changes, the image filenames change too.)

File: _posts/2022-01-17-cockpit-261.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+![screenshot of show busiest cpu core]({{ site.baseurl }}/images/261-metrics-show-busiest-cpu-core.png)
+
+## Storage: Unmounting or deleting a busy filesystem is now supported
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit/pull/16431>
+{:.note}
+
+Cockpit can now stop processes and services that keep a filesystem busy before unmounting or destroying a filesystem.
+
+This is accompanied by a general improvement in how destructive actions are summarized in the their dialogs.
```
Comment (allisonkarlitskaya) [inline _posts/2022-01-17-cockpit-261.md:33]:
```suggestion
This is accompanied by a general improvement in how destructive actions are summarized in their dialogs.
```

File: _posts/2022-01-17-cockpit-261.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+
+## Podman: Add pause/resume to containers
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit-podman/pull/870>
+{:.note}
+
+Bring containers to the similiar feature parity to podman pods by adding
+pause/resume options. Also make the order of the menu options the same
+for pods and containers and add "Force restart" to the pod actions.
```
Comment (allisonkarlitskaya) [inline _posts/2022-01-17-cockpit-261.md:70]:
```suggestion
Add the pause and resume options — already available for pods — to
individual containers. Also make the order of the menu options the same
for pods and containers and add "Force restart" to the pod actions.
```

File: _posts/2022-01-17-cockpit-261.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+## Always pull the latest image when creating a new container
+
+State: **OPEN** (will not be included) @ <https://github.com/cockpit-project/cockpit-podman/pull/859>
+{:.warn}
+
+When a local image `fedora:latest` is available but a newer version is available on the image registry, checking the "pull latest image" checkbox will pull the latest version before creating the new container
```
Comment (allisonkarlitskaya) [inline _posts/2022-01-17-cockpit-261.md:77]:
```suggestion
When the `:latest` tag of an image is available locally, but a newer version is available on the image registry, checking the "pull latest image" checkbox will now pull the latest version before creating the new container
```

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]

File: _posts/2022-01-17-cockpit-261.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/261)
+* [Cockpit Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit)
+* [Cockpit Fedora 34](https://bodhi.fedoraproject.org/updates/?releases=F34&packages=cockpit)
+* [cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit-podman/releases/tag/40)
+* [cockpit-podman Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit-podman)
+* [cockpit-podman Fedora 34](https://bodhi.fedoraproject.org/updates/?releases=F34&packages=cockpit-podman)
+
+*[CPU]: Central Processing Unit, the "brain" of a computer
```
Comment (allisonkarlitskaya) [inline _posts/2022-01-17-cockpit-261.md:95]: also: lol.  seriously?  I don't think our target audience needs this if we're freely discussing pods, containers, image tags, unmounting busy filesystems, etc. all without further explanation.

**Comment by marusak** [comment]
Thanks @allisonkarlitskaya for the review. I've updated the release notes themselves.

File: _posts/2022-01-25-cockpit-261.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+![screenshot of unmounting or deleting a busy filesystem is now supported]({{ site.baseurl }}/images/261-storage-unmounting-or-deleting-a-busy-filesystem-is-now-supported-2.png)
+
+
+## Podman: Add pause/resume to containers
+
+Add the pause and resume options — already available for pods — to
+individual containers. Also make the order of the menu options the same
+for pods and containers and add "Force restart" to the pod actions.
+
+## Add option to pull the latest image when creating a new container
```
Comment (garrett) [inline _posts/2022-01-25-cockpit-261.md:34]:
```suggestion
## Podman: Add option to pull the latest image when creating a new container
```

This is lacking the "Podman" prefix.

**Comment by garrett** [comment]
I've generated the release notes from the PRs. I guess it's time to polish up directly in the PR and we'll want to squash things down before/while merging.

First mod: Added Podman to one of the headings. 

We still need summary, links to be updated, etc.

**Comment by marusak** [comment]
Added summary + links. Ready for review.

**Review by martinpitt — APPROVED** [review]
Thanks!

**Pull request merged**